cassandra/storage/src/main/java/org/opennms/newts/persistence/cassandra/CassandraSampleRepository.java

```diff
@@ -249,6 +261,35 @@ public void insert(Collection<Sample> samples, boolean calculateTimeToLive) {
 
     }
 
+    @Override
+    public void delete(Context context, Resource resource) {
+        final Timestamp start = Timestamp.now().minus(m_ttl, TimeUnit.SECONDS);
```

Comment:
m_ttl can be 0 (when rows never expire), what's the expected behavior in this case?